app/models/authentication_provider.rb

```diff
@@ -31,7 +31,7 @@ class AuthenticationProvider < ApplicationRecord
 
   validates :client_id, :client_secret, presence: true, if: :oauth_config_required?
 
-  with_options format: { with: URI.regexp(%w(http https)), allow_blank: true, message: :invalid_url } do |ops|
+  with_options format: { with: URI::DEFAULT_PARSER.make_regexp(%w[http https]), allow_blank: true, message: :invalid_url } do |ops|
```

Comment:
This and other similar changes were just made due to deprecation of `URI.regexp`. The new code does not have any functional difference.